index.ts

```diff
@@ -32,6 +35,30 @@ function toCollectionName(name: string) {
     .slice(0, 32)
 }
 
+const openSearchApiFile = 'openSearchApi.js'
+
+async function call_API(cwd: string) {
+  //Load api call file and run all api calls to cluster
+  const api_path = join(cwd, openSearchApiFile)
+  let result
+  if (await exists(api_path)) {
+    console.log(`Found ${openSearchApiFile} file, deploying ML model`)
+    result = (await import(pathToFileURL(api_path).toString())).default
+
+    //result should be a function that returns a promise
+    if (typeof result === 'function') {
+      result = result()
+    }
+
+    //wait for the returned promise to resolve and thus all api calls to complete
+    if (result instanceof Promise) {
+      await result
+    }
+  } else {
+    console.log(`No ${openSearchApiFile} file found.`)
+  }
```

Comment:
Remove this log message, since this is not an unexpected occurrence.

```suggestion
  }
```